timber.go

```diff
@@ -500,135 +522,172 @@ func (t *Timber) prepare(lvl Level, msg string, depth int) *LogRecord {
 		MethodPath:  methodPath,
 		PackagePath: packagePath,
 		HostName:    hostname,
+		Extra:       extra,
 	}
 }
 
+var emptyExtra map[string]string
```

Comment:
Arguably a new `prepareAndSendEx` that `*Ex` methods invoke is preferable here.

Reply:
Updated in https://github.com/cocoonlife/timber/pull/1/commits/1c84e0cf858e0e1a919bedc96de6948247935c70